submissions/israelOladosu/DomManipulation/index.js

```diff
@@ -0,0 +1,53 @@
+let submitForm = document.getElementById("validateForm"),
+  qty = document.getElementById("qty"),
+  shipping_cost = document.getElementById("shipping_cost"),
+  total_cost = document.getElementById("total_cost"),
+  price = document.getElementById("price");
+
+const RegEx = /^[0-9]$/;
+
+function shippingCost(cost) {
+  if (+cost < 1000) {
+    shipping_cost.lastChild.innerHTML = `100`;
+  } else if (+cost >= 1000 && +cost < 20000) {
+    shipping_cost.lastChild.innerHTML = `${
+      (15 / 100) * +price.lastChild.innerText
+    }`;
+  } else if (+cost >= 2000 && +cost < 100000) {
+    shipping_cost.lastChild.innerHTML = `${
+      (20 / 100) * +price.lastChild.innerText
+    }`;
+  } else {
+    shipping_cost.lastChild.innerHTML = `25000`;
+  }
+}
+
+qty.addEventListener("input", function () {
+  if (+qty.value > 1) {
+    console.dir(price.lastChild);
+    price.lastChild.innerHTML = `${+qty.value * +price.lastChild.innerText}`;
```

Comment:
Meanwhile. why do you keep using `lastChild`? Are they many elements which the `id = "price"`? Same goes for others too

Reply:
Yes, i was targeting the span element inside them, the span element is the one holding the figure... I was trying not to touch the currency sign that proceeds the span,

Reply:
Not yes as in many elements using `id = "price"`. That was a i mistake, I just meant to say i was targeting the individual elements span